MapboxNavigation/RouteVoiceController.swift

```diff
@@ -193,6 +191,9 @@ open class RouteVoiceController: NSObject, AVSpeechSynthesizerDelegate {
      - parameter ignoreProgress: A `Bool` that indicates if the routeProgress is added to the instruction.
      */
     open func speak(_ instruction: SpokenInstruction, with locale: Locale?, ignoreProgress: Bool = false) {
+        // Don't require background audio unless the implementer is actually using speech APIs
+        self.verifyBackgroundAudio()
```

Comment:
Note that this ultimately calls an `assertionFailure` so it shouldn't have any bearing on release builds one way or the other.